src/main/java/org/fundacionjala/pivotal/core/e2e/SauceTest.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.pivotal.core.e2e;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.URL;
+
+public class SauceTest {
+    private static final String USERNAME = "SAUCE_USERNAME";
+    private static final String ACCESS_KEY = "SAUCE_ACCESS_KEY";
+    private static final String URL = "https://" + USERNAME + ":" + ACCESS_KEY + "@ondemand.saucelabs.com:443/wd/hub";
+
+    public static void createDriver(String browser, String version,String os) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)